src/cgr_gwas_qc/workflow/scripts/concordance_table.py

```diff
@@ -28,8 +33,8 @@
 app = typer.Typer(add_completion=False)
 
 DTYPES = {
-    "ID1": "string",
-    "ID2": "string",
+    "ID1": "category",
+    "ID2": "category",
```

Comment:
Why do we convert these to "category"?

Reply:
For repetitive values like IDs in this case categorical dtype is more memory efficient than string. The orignal read function is reading as string because we need to sort alphanumerically between columns which requires string dtype.

Reply:
It's true that switching to a categorical dtype can save memory **when the string variable consists of a limited number of repetitive values**. But in our case, wouldn't each ID be unique? Then using the categorical dtype may actually consume more memory. According to the pandas documentation: 
> If the number of categories approaches the length of the data, the Categorical will use nearly the same or more memory than an equivalent object dtype representation.

[ref](https://pandas.pydata.org/pandas-docs/stable/user_guide/categorical.html#categorical-memory)

Reply:
It's true that switching to a categorical dtype can save memory **when the string variable consists of a limited number of repetitive values**. But in our case, wouldn't each ID be unique? Then using the categorical dtype may actually consume more memory. According to the pandas documentation: 
> If the number of categories approaches the length of the data, the Categorical will use nearly the same or more memory than an equivalent object dtype representation.

[ref](https://pandas.pydata.org/pandas-docs/stable/user_guide/categorical.html#categorical-memory)

Reply:
Hi @jaamarks , thanks for taking the review. The IDs aren't unique here and which makes sense because these are pair wise distances so the format is something like this:

> ID1,ID2
> 201113910004_R01C01,201113910004_R03C02
> 201113910004_R01C01,201113910004_R04C02
> 201113910004_R01C01,201113910004_R05C02
> 201113910004_R01C01,201113910004_R06C02

Take look at the sample_level/concordance/plink.csv. Feel free to let me know if I am missing something here and I will revert it to string. Thank you.

Reply:
Oh I see - good point. In this case I also think it makes sense to have as categorical dtype. Thanks!